### PR TITLE
set password for docker run

### PR DESCRIPTION
**Pull request by FxKu**
postgres docker image now requires to define a password or set auth method to trust.
See [issue](https://github.com/docker-library/postgres/issues/681)

### CODE REVIEW

File: extension/tests/container.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ PG_VERSION=$1
 TESTCMD=$2
 
 docker build -t pgmemento-test--$PG_VERSION - < ./Dockerfile-$PG_VERSION;
-CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION);
+CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION -e POSTGRES_PASSWORD=password);
```
Comment (dnsl48) [inline extension/tests/container.sh]:
```suggestion
CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento -e POSTGRES_PASSWORD=password pgmemento-test--$PG_VERSION);
```

The parameters after the image name would be interpreted as extra arguments for the command run within the container. In this case we want to pass this parameter to the docker itself, so it should go before the image name (whir is `pgmemento-test--$PG_VERSION` in our case)

Comment (FxKu) [reply extension/tests/container.sh]: thx. also found this myself by now :smiley: 
lets see

File: extension/tests/container.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ PG_VERSION=$1
 TESTCMD=$2
 
 docker build -t pgmemento-test--$PG_VERSION - < ./Dockerfile-$PG_VERSION;
-CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION);
+CONTAINER=$(docker run -d -e POSTGRES_PASSWORD=password) -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION;
```
Comment (dnsl48) [inline extension/tests/container.sh]:
```suggestion
CONTAINER=$(docker run -d -e POSTGRES_PASSWORD=password -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION);
```

Comment (dnsl48) [reply extension/tests/container.sh]: another typo :D

**Pull request merged**